_posts/2018-12-22-Reading-Effective-use-of-typescript-with-react.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Reading: Best practices for using Typescript with React"
+date:   2018-12-22 10:00:00 +0800
+categories: readingnote
+author: Rukai Yu
+---
+
+[Article Link](https://www.allthingsdistributed.com/2006/11/working_backwards.html)
```

Comment:
Are you sure you linked the right article?

Reply:
ahahahha,,,, fixed.